test/choosetests.jl

```diff
@@ -149,6 +149,7 @@ function choosetests(choices = [])
     filtertests!(tests, "compiler/EscapeAnalysis", [
         "compiler/EscapeAnalysis/local", "compiler/EscapeAnalysis/interprocedural"])
     filtertests!(tests, "stdlib", STDLIBS)
+    filtertests!(tests, "internet_required", ["download", "Downloads", "LibGit2/online", "Pkg"])
```

Comment:
In https://github.com/NixOS/nixpkgs/pull/205584#issuecomment-1345574283 we also have Artifacts, LazyArtifacts and LibCURL failures. Maybe we should share the list with `net_required_for` just below?

Reply:
Done. Although I don't think Sockets actually needs Internet access, so I kept that one in. Apart from Sockets, though, the two lists are exactly the same now.